### PR TITLE
Tooltip redesign

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-10574
https://ag-grid.atlassian.net/browse/AG-13524

### CODE REVIEW

File: packages/ag-charts-community/src/chart/interaction/tooltipManager.css
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/interaction/tooltipManager.css:1]: Probably one for @markdurrant to review

File: packages/ag-charts-types/src/series/cartesian/boxPlotOptions.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export interface AgBoxPlotSeriesTooltipRendererParams<TDatum>
     extends BoxPlotOptionsKeys,
         BoxPlotOptionsNames,
         AgSeriesTooltipRendererParams<TDatum> {
-    fill?: CssColor;
+    // @todo(AG-10574) fill?: CssColor;
```
Comment (alantreadway) [inline packages/ag-charts-types/src/series/cartesian/boxPlotOptions.ts:53]: ?

File: packages/ag-charts-types/src/series/cartesian/candlestickOptions.ts [hunk 2]
```diff
@@ -19,7 +18,7 @@ export interface AgCandlestickSeriesItemStylerParams<TDatum>
 export interface AgCandlestickSeriesTooltipRendererParams<TDatum>
     extends AgOhlcSeriesBaseTooltipRendererParams<TDatum>,
         AgOhlcSeriesBaseOptions {
-    fill?: CssColor;
+    // @todo(AG-10574) fill?: CssColor;
```
Comment (alantreadway) [inline packages/ag-charts-types/src/series/cartesian/candlestickOptions.ts:21]: ?

File: packages/ag-charts-types/src/series/cartesian/ohlcOptions.ts [hunk 3]
```diff
@@ -14,7 +13,7 @@ export type AgOhlcSeriesItemStylerParams<TDatum> = AgOhlcSeriesBaseItemStylerPar
 export interface AgOhlcSeriesTooltipRendererParams<TDatum>
     extends AgOhlcSeriesBaseTooltipRendererParams<TDatum>,
         AgOhlcSeriesBaseOptions {
-    stroke?: CssColor;
+    // @todo(AG-10574) stroke?: CssColor;
```
Comment (alantreadway) [inline packages/ag-charts-types/src/series/cartesian/ohlcOptions.ts:16]: ?

File: packages/ag-charts-types/src/series/hierarchy/sunburstOptions.ts [hunk 4]
```diff
@@ -16,15 +16,15 @@ export type AgSunburstSeriesLabelHighlightOptions<TDatum> = Pick<
...
 export interface AgSunburstSeriesTooltipRendererParams<TDatum>
-    extends AgChartCallbackParams<TDatum>,
+    extends AgSeriesTooltipRendererParams<TDatum>,
         AgSunburstSeriesOptionsKeys,
         AgSunburstSeriesOptionsNames {
     /** The depth of the datum in the hierarchy. */
     depth: number;
     /** The title of the Sunburst segment. */
-    title?: string;
+    // @todo(AG-10574) title?: string;
```
Comment (alantreadway) [inline packages/ag-charts-types/src/series/hierarchy/sunburstOptions.ts:25]: ?

File: packages/ag-charts-website/src/content/docs/benchmarks/_examples/summary/main.ts [hunk 5]
```diff
@@ -201,9 +201,9 @@ function generatePerformanceChart(keyX: number, keyY: number) {
                 stackGroup: 'time',
                 tooltip: {
                     enabled: true,
-                    renderer: ({ datum }: AgCartesianSeriesTooltipRendererParams<Suite>) => ({
-                        content: formatMillis(datum.results[testName]!.timeMs, 2),
-                    }),
+                    // renderer: ({ datum }: AgCartesianSeriesTooltipRendererParams<Suite>) => ({
```
Comment (alantreadway) [inline packages/ag-charts-website/src/content/docs/benchmarks/_examples/summary/main.ts]: ?

File: packages/ag-charts-website/src/content/gallery/_examples/multiple-box-plots/main.ts [hunk 6]
```diff
@@ -75,14 +45,14 @@ const options: AgChartOptions = {
             type: 'box-plot',
             data: data['Jan - Mar 2023'],
             yName: 'Jan - Mar 2023',
-            tooltip,
+            // tooltip,
```
Comment (alantreadway) [inline packages/ag-charts-website/src/content/gallery/_examples/multiple-box-plots/main.ts]: ?

File: packages/ag-charts-website/src/content/docs/benchmarks/_examples/summary/main.ts (hunk 5 above)
Comment (jacobp100) [reply packages/ag-charts-website/src/content/docs/benchmarks/_examples/summary/main.ts]: Updated

File: packages/ag-charts-website/src/content/gallery/_examples/multiple-box-plots/main.ts (hunk 6 above)
Comment (jacobp100) [reply packages/ag-charts-website/src/content/gallery/_examples/multiple-box-plots/main.ts]: Fixed

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**